src/demo.js

```diff
@@ -88,9 +92,24 @@ export default class DemoTimeline extends Component {
       }
     }
 
+    const dataGridColumns = [
+      {
+        width: 100,
+        headerKey: 1
+      },
+      {
+        width: 250,
+        dataKey: 'description',
+        headerKey: 2,
+        headerRenderer: () => {
```

Comment:
Accepts a "generic" React component? Which may be a function, like here, or a class component? Same thing for `cellRenderer`. HR doesn't need to have anything in the props. But for CR, let's look at our other table, to see what format it uses. I'd like those HR + CR to be compatible here.

Let's also have a `headerLabel`; the "easy" mode equivalent of `labelProperty`.